cluster/gce/container-linux/configure-helper.sh

```diff
@@ -294,6 +296,7 @@ users:
 clusters:
 - name: local
   cluster:
+    server: https://${apiserver_address}
```

Comment:
pass in the full address, including `https://` and any required port (applies in general, hoist the scheme and ports as high up as possible)